client/reader/_style.scss

```diff
@@ -17,6 +17,22 @@
 	}
 }
 
+.reader__content,
+.reader__full-post-content {
+	sup, sub {
+		position: relative;
+		font-size: 0.83em;
```

Comment:
I think we usually like to use `px` units, but I can see how this would be better for `sup` and `sub` in headings and other things that don't use the usual body px size. @shaunandrews thoughts?